attributes/default.rb

```diff
@@ -638,6 +638,8 @@
 default['cluster']['realmemory_to_ec2memory_ratio'] = 0.95
 default['cluster']['slurm_node_reg_mem_percent'] = 75
 default['cluster']['slurmdbd_response_retries'] = 30
+default['cluster']['slurm_console_logging']['default']['enabled'] = 'True'
```

Comment:
question: Could you explain the benefit of using multiple nested dict here to store the default value and enable? What about simply use `default['cluster']["compute_console_logging_max_sample_size"]` and `default['cluster']["compute_console_logging_enabled"]` because the clustermgtd configuration is also not nested?

Reply:
Should it be enable only when https://docs.aws.amazon.com/parallelcluster/latest/ug/Monitoring-v3.html#yaml-Monitoring-Logs-CloudWatch-Enabled is enable?

Reply:
It will use Monitoring-Logs-CloudWatch-Enabled if it is present, otherwise it defaults to True.

Reply:
As to the first question, to me it seems more readable and kind of groups things by purpose, but I'm not too opinionated on this.

Reply:
removed ['default'] sub-tree.